src/app/pages/login/login.page.ts

```diff
@@ -84,6 +83,8 @@ export class LoginPage extends BasePageComponent implements OnInit {
         await this.authenticationProvider.logout();
       }
       this.appInitError = error;
+
+      // TODO: Dispatch action and log the error elsewhere not to the console.
       console.log(error);
```

Comment:
console.log and TODO, should these still be here?

Reply:
One of them has to stay to remind us to save and send errors in a logs action once the logging mechanism is implemented. I'll remove the `console.log`